### PR TITLE
feat: add documentation for new usage page

### PR DESCRIPTION
**Pull request by jfhinchcliffe**
we're about to launch a new usage page for job minutes and test executions. this PR adds the documentation for the new page.

There isn't an overarching section for information on both Job Minutes and Test Analytics, so I've duplicated thigns a bit across each section.

The job minutes / usage page has been updated, and a new test analytics page added which is title 'Test Executions'. Could be convinced that this should go somewhere else, but seemed logical enough for a start.

### CODE REVIEW

**Comment by jfhinchcliffe** [comment]
@mbelton-buildkite @jameshill when you have time for a gander the staging server has these pages ready at:
https://bk-docs-pr-1956.onrender.com/docs/pipelines/job-minutes
https://bk-docs-pr-1956.onrender.com/docs/test-analytics/test-executions

File: pages/test_analytics/test_executions.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Test executions
+
+Each [Buildkite plan](https://buildkite.com/pricing) has test execution inclusions, which vary depending on the plan type and the number of users in your organization.
+
+You can find the test execution count on each run page
```
Comment (mbelton-buildkite) [inline pages/test_analytics/test_executions.md:5]: The sentence needs to end with either a period or colon. To make it more parallel with the Pipelines page, perhaps:

```suggestion
You can find the test execution details for a run at the bottom of the run page, and your organization's [total usage](#usage-page) in Settings.
```

File: pages/test_analytics/test_executions.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<%= image "test_executions.png", alt: "Test executions run page" %>
+
+## Usage page
+
+The [usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows a breakdown of job minutes and test executions for your organization.
```
Comment (mbelton-buildkite) [inline pages/test_analytics/test_executions.md:10]: As with the other page:

```suggestion
The [Usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows a breakdown of job minutes and test executions for your organization.
```

File: pages/test_analytics/test_executions.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+<%= image "usage_page.png", alt: "Usage summary page" %>
```
Comment (mbelton-buildkite) [inline pages/test_analytics/test_executions.md:12]: Should we remove this image?

File: pages/test_analytics/test_executions.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+The [test executions usage page](https://buildkite.com/organizations/~/usage/test_executions) graphs the total executions over the organization's current billing period. It includes a breakdown of the current usage by suite as well as a CSV download of usage over the period.
+
+<%= image "test_executions_usage_page.png", alt: "Test executions usage summary page" %>
```
Comment (mbelton-buildkite) [inline pages/test_analytics/test_executions.md:16]: Should we remove this image?

File: pages/test_analytics/test_executions.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+You can find the test execution count on each run page
+<%= image "test_executions.png", alt: "Test executions run page" %>
+
+## Usage page
+
+The [usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows a breakdown of job minutes and test executions for your organization.
+
+<%= image "usage_page.png", alt: "Usage summary page" %>
+
+The [test executions usage page](https://buildkite.com/organizations/~/usage/test_executions) graphs the total executions over the organization's current billing period. It includes a breakdown of the current usage by suite as well as a CSV download of usage over the period.
```
Comment (mbelton-buildkite) [inline pages/test_analytics/test_executions.md:14]: As with the other page:

```suggestion
The [Test executions usage page](https://buildkite.com/organizations/~/usage/test_executions) graphs the total executions over the organization's current billing period. It includes a breakdown of the current usage by suite and a CSV download of usage over the period.
```

File: pages/pipelines/job_minutes.md [hunk 6]
```diff
@@ -17,9 +17,16 @@ Parallelism does not affect how job minutes are calculated, the following situat
 
 ## Usage page
 
-The [Usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows your plan information, any applicable inclusions or limits, and recent job minute usage.
+The [usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows a breakdown of job minutes and test executions for your organization.
 
-<%= image "usage.png", width: 554, height: 794, alt: "Job minute usage page" %>
+<%= image "usage_page.png", alt: "Usage summary page" %>
+
+The [job minutes usage page](https://buildkite.com/organizations/~/usage/job_minutes) graphs the total job minute usage over the organization's current billing period. It includes a breakdown of the current usage by pipeline as well as a CSV download of usage over the period.
```
[the remark this answers is not in the document]
Comment (mbelton-buildkite) [reply pages/pipelines/job_minutes.md:24]: This also emphasizes the current billing period, but it looks like it also provides information about previous billing periods. Is that right? If so, consider including that.

**Review by mbelton-buildkite — COMMENTED** [review]
I've left a bunch of comments, but there's nothing major. Mainly wondering whether it would be good to cut down on the screenshots. 🤔

File: pages/pipelines/job_minutes.md [hunk 7]
```diff
@@ -17,9 +17,16 @@ Parallelism does not affect how job minutes are calculated, the following situat
 
 ## Usage page
 
-The [Usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows your plan information, any applicable inclusions or limits, and recent job minute usage.
+The [usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows a breakdown of job minutes and test executions for your organization.
 
-<%= image "usage.png", width: 554, height: 794, alt: "Job minute usage page" %>
+<%= image "usage_page.png", alt: "Usage summary page" %>
```
[the remark this answers is not in the document]
Comment (jfhinchcliffe) [reply pages/pipelines/job_minutes.md:22]: No worries - I'll remove from the `job_minutes` and `test_executions` pages.

File: pages/pipelines/job_minutes.md [hunk 8]
```diff
@@ -17,9 +17,16 @@ Parallelism does not affect how job minutes are calculated, the following situat
 
 ## Usage page
 
-The [Usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows your plan information, any applicable inclusions or limits, and recent job minute usage.
+The [usage page](https://buildkite.com/organizations/~/usage) is available on every Buildkite plan and shows a breakdown of job minutes and test executions for your organization.
```
[the remark this answers is not in the document]
Comment (jfhinchcliffe) [reply pages/pipelines/job_minutes.md:20]: Updated ✨

File: pages/pipelines/job_minutes.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jfhinchcliffe) [reply pages/pipelines/job_minutes.md:24]: Yeah, previous and current. I've amended it to just be 'billing periods'

File: pages/test_analytics/test_executions.md (hunk 4 above)
Comment (jfhinchcliffe) [reply pages/test_analytics/test_executions.md:16]: looks fine without it ✅

**Comment by jfhinchcliffe** [comment]
@mbelton-buildkite Thanks for the review! I've amended the commit and re-pushed 🥳

**Review by jameshill — APPROVED** [review]
We've enabled Usage Breakdown for all orgs

**Comment by mbelton-buildkite** [comment]
I just came to review this after your updates but I can see that it's been merged before approval. What happened there?

I noticed a couple of things that still need to be addressed:

- A few images were merged in that are not used in the doc. These should be deleted.
- The nav entry should use sentence case: Test Executions -> Test executions.

Can you please follow up with a PR to fix those?

**Comment by jfhinchcliffe** [comment]
Heya @mbelton-buildkite . Sorry about that - we set the feature live earlier today on Buildkite and jumped the gun a bit getting this merged.
I've put up a cleanup PR here for the issues you mentioned: https://github.com/buildkite/docs/pull/1969

**Pull request merged**